test/testGear.cc

```diff
@@ -0,0 +1,17 @@
+#include "opengl/GrailGUI.hh"
+#include "CAD/Gear.hh"
+using namespace std;
+using namespace grail;
+
+class TestGear : public GLWin {
+ public:
+  TestGear() : GLWin(0x000000, 0xCCCCCC, "Test Gear") {}
+  void init() {
+    MainCanvas* c = currentTab()->getMainCanvas();
+    const Style* s = getDefaultStyle();
+    Gear* circ = c->addLayer(new Gear(c, 500.0, 500.0, 100.0, 100.0, 6.0, 100.0, 100.0, s));    
```

Comment:
This has a funky glitch where the first and last point are connecting to 0,0. This is definitely not the fault of anyone on StevensCAD team though, because this is an issue that we ended up fixing in a different branch a while ago. Nice work.

Reply:
Correction, this is not using multishape, so its something we'll have to diagnose later.